UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowDailyFiles.cshtml

```diff
@@ -14,12 +14,23 @@
 
 <div class="tab-content" id="nmTabContent">
     <div class="tab-pane fade show active" id="daily" role="tabpanel" aria-labelledby="daily-tab">
+          @if(@ViewBag.IsDistributor)
+    {
+        <div id="NMInfo" class="alert alert-primary p-4" role="alert">
+    <i class="bi bi-exclamation-circle-fill" aria-hidden="true"></i>
+    <div class="alert-head">Temporary Notice</div>
+    <p>
+        Redirecting to download the Daily tracing file due to unavailability in MSI at the moment : <br>
+        <td><a href="~/nms/tracings/Title_and_Index_2022.pdf" target="_blank">Daily Tracings File</a></td>
```

Comment:
Please remove <td> tag as its not required here. Element 'td' cannot be nested inside element 'p'.
Also, please correct the alignment for the whole @if block.

Reply:
Removed and alignment made proper